services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1302,6 +1303,68 @@ void GraphBuilderOrt::AddReduceOperation(const mojom::Reduce& reduce) {
                          attributes);
 }
 
+void GraphBuilderOrt::AddResample2dOperation(
+    const mojom::Resample2d& resample2d) {
+  const std::string node_name = GetNodeName(resample2d.label);
+  const std::string input_name = GetOperandName(resample2d.input_operand_id);
+  const std::string output_name = GetOperandName(resample2d.output_operand_id);
+  const std::vector<uint32_t>& output_shape =
+      GetOperand(resample2d.output_operand_id).descriptor.shape();
+  std::vector<const char*> input_names = {input_name.c_str()};
+
+  // ROI only takes effect when coordinate_transformation_mode is
+  // “tf_crop_and_resize”. Currently, WebNN only supports "half_pixel".
```

Comment:
Nit: it would be good to also explain ONNX Resize op's attribute `coordinate_transformation_mode` is default to `half_pixel`.

Reply:
Done. Please take another look. Thanks!